routes/dashboard.rb

```diff
@@ -1,15 +1,34 @@
 class App < Sinatra::Base 
   get '/' do
```

Comment:
Consider extracting all the logic to PORO objects
http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/